docs/index.md

```diff
@@ -98,7 +98,7 @@ or as a development dependency for your project:
 $ npm install --save-dev mocha
 ```
 
-> As of v6.0.0, Mocha requires Node.js v8.0.0 or newer.
+> As of v6.0.0, Mocha requires Node.js v7.0.0 or newer.
```

Comment:
Is this correct? Should be:
"As of v7.0.0, Mocha requires Node.js v8.0.0 or newer."
Next major Mocha release will be v7. And Node will be supported >=v8.